### ISSUE TITLE
Mischievous Chips

### ISSUE THREAD

**Issue by tsop14** [guardian/grid#1801]
😈🍟

- [ ] Double-clicking to select all doesn't work if you do it in the are after the word.
- [ ] When try to select text with a broad click-drag, depending on where do you start dragging (still: always within the text box area), different glitches happen in different browsers... 
- [ ] Cannot Paste (from the context menu).

**Comment by paperboyo** [comment]
[] :fries: do not play well with `usages@` syntax. `usages` falls out of a chip that only says e.g. `status:` and `@` is hidden and... just have a play :)

**Comment by paperboyo** [comment]
- [ ] :fries: bleed out of the searchbox on latest beta and nightly of Firefox on Win10 like so:

![chips bleed](https://cloud.githubusercontent.com/assets/6032869/13557544/5c780728-e3ea-11e5-824b-cf42220ff2c8.PNG)

**Comment by NickPapacostas** [comment]
:fries: they're trying to be clever with nested queries
- [ ] query limited to size of chip when it aint a chip!
(can't reproduce, just keep an eye out)

- [ ] when I do a nested query, click on an image, and then go back to search, chips kick in incorrectly
the query in the example below was "usages@>added:2015"
![screen shot 2016-03-07 at 12 27 40](https://cloud.githubusercontent.com/assets/1441582/13569460/1208b892-e460-11e5-87da-fe9cbeeb2ca9.png)

**Comment by paperboyo** [comment]
- [ ] :fries: embarrassingly reveal the lack of UI responsiveness when cut long enough:

![chips-long](https://cloud.githubusercontent.com/assets/6032869/14135680/c88e8a12-f656-11e5-9c6f-04d45d5738ce.png)

**Comment by paperboyo** [comment]
- [ ] Trying to add a `#` at the beginning of the search term (sound effects are: aaarrrrgh!! and nooooo!):

![untitled](https://cloud.githubusercontent.com/assets/6032869/14210610/a65cb168-f821-11e5-8a65-9d2bfe20540c.gif)

**Comment by tsop14** [comment]
Think  #1830 will fix this?

**Comment by paperboyo** [comment]
Oh cool, haven't connected the dots, sorry! Can I have a play with crop, pretty please (at any point, incl. when it's in PROD ;-) ).

**Comment by tsop14** [comment]
Yep, the branch will do this 👉 ![add label](https://cloud.githubusercontent.com/assets/8484757/14210787/7cf15d1e-f822-11e5-852e-8f61f1093dec.gif)

**Comment by paperboyo** [comment]
- [ ] Collections are not suggested. Also while trying to force them into a chip - my chip died. A dead :fries:

![dead chip](https://cloud.githubusercontent.com/assets/6032869/14353972/55d5b206-fcd3-11e5-944d-bca1974d6348.gif)

**Comment by paperboyo** [comment]
- [ ] :fries: duplicate search term after pressing `:`

![chips colon](https://cloud.githubusercontent.com/assets/6032869/14590655/31f228d8-04f8-11e6-9695-9ce5b3679004.gif)

P.s. I’m not against :fries:! I love :fries:. It’s a pity they do always not love me back...

**Comment by paperboyo** [comment]
Make them work like in PFI where typing + and - does not immediately launch a chip preventing one from running NOT queries.

**Comment by craigh04** [comment]
You cannot manually type the name of the filter followed by a colon, eg title:
nor can you select +t  and get offered this search on the dropdown relating to chips starting with t.
Nor does title appear on the hint, listing 
Currently the only place you can start a search against a title text is from the info panel ,when the image has a title text.
